designer/client/reducers/component/componentReducer.options.ts

```diff
@@ -151,5 +151,13 @@ export function optionsReducer(state, action: OptionsActions) {
           options: { ...options, allowPrePopulation: payload },
         },
       };
+    case Options.EDIT_OPTIONS_DISABLE_CHANGING_FROM_SUMMARY:
+      return {
```

Comment:
Every day I wonder why react is the defacto component library, and why this is (was?) official guidance